### ISSUE TITLE
Settings option "Ignore unsaved" is not understandable

### ISSUE THREAD

**Issue by MathJud** [dakhnod/Meshenger#16]
Please add an understandable description such as: block calls from unknown devices

**Comment by mwarning** [comment]
Yes, this was horrible. It was renamed. Should be in the new release: https://github.com/meshenger-app/meshenger-android/releases/tag/v2.0.0
(but now the option does not serve a purpose - the unknown caller feature needs to be added in future releases)
